packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -344,15 +344,15 @@ Use the text editor to paste the snippet below in your WordPress page
 
 Replace assets/ with the path from your Media Library 
 
-        ---- WP Deployment SNIPPET ----
+        ---- WordPress Deployment SNIPPET ----
```

Comment:
Now that the manually instructions for installing the plugin are available, you can add a link to them here, and with that done, you can re-enable including this block in the generated README.txt.